torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -1008,8 +1009,7 @@ def _linear_bf16_act_uint4_weight_check(input_tensor, weight_tensor, bias):
         _aqt_is_uint4(weight_tensor) and
         weight_tensor.dtype == torch.bfloat16 and
         len(weight_tensor.shape) == 2 and
-        weight_tensor.zero_point_domain == ZeroPointDomain.FLOAT and
-        isinstance(weight_tensor.layout_type, TensorCoreTiledLayoutType)
+        weight_tensor.zero_point_domain == ZeroPointDomain.FLOAT
```

Comment:
does the impl work with PlainLayoutType?

Reply:
yea can revert